db/stats.py

```diff
@@ -10,23 +10,30 @@
 
 
 def get_last_submitted_recordings():
-    last_submitted_data = db.cache.get('last-submitted-data')
-    if not last_submitted_data:
+    cache_key = "last-submitted-recordings"
+    last_submissions = db.cache.get(cache_key)
+    if not last_submissions:
         with db.engine.connect() as connection:
-            result = connection.execute("""SELECT mbid,
-                                     data->'metadata'->'tags'->'artist'->>0,
-                                     data->'metadata'->'tags'->'title'->>0
-                                FROM lowlevel
-                            ORDER BY id DESC
-                               LIMIT 5
-                              OFFSET 10""")
```

Comment:
because the highlevel computation might not have finished yet. We want to show 5 that we are pretty sure are complete before someone clicks on the link. We could instead join to highlevel